### ISSUE TITLE
Depowered doors can't be closed.

### ISSUE THREAD

**Issue by thebadman4662** [space-wizards/space-station-14#33795]
## Description
So you get nearly instant spacing of whole station if engi can't prevent any kind of brownouts from the start of round.

**Reproduction**
Simply crowbar depowered door.

**Screenshots**
![depowered doors can't be closed without power](https://github.com/user-attachments/assets/df9861e2-add0-418e-a369-17854536a487)

**Comment by Ian321** [comment]
Replicated on Leviathan 73152.

https://github.com/user-attachments/assets/8128897c-974b-4d36-85e4-a018412eb466

**Comment by zHonys** [comment]
I found the issue, trying to make a fix

**Comment by FairlySadPanda** [comment]
The bug is basically that prying an unpowered AirlockComponent door open doesn't hit a specific "can change state" function, but when prying it closed it does, and the code in question only works if the door is powered. So specifically an Airlock can't be pried closed.

This smells like cascading rot elsewhere impacting badly-architected code here. I'll see if I can fix this in a way that simplfies the system, but on the face of it the easiest fix is to just get rid of the "is powered" check when attempting to close an airlock.

**Comment by FairlySadPanda** [comment]
Yeah this is basically because there's two reasons a door can close:

1. For reason of power
2. By reason of pry

But there's no handling for recognizing the difference between these two states. Door code is complex and this particular route has been missed.

This needs encapsulating by extending the states a door can be in to prevent this modelling drift. A door can be closing under its own power (Closing) or by an external factor that does not require power (ForceClosing). Encapsulating this will allow for code routes that make sense.

**Comment by zHonys** [comment]
I already fixed the main issue of: the door can't be closed when de powered, some hours ago. But, there already was an bug that went undetected that makes the closing animation not play correctly, because it goes in the Closing -> Open -> Closed, i'll make a PR with the basic fix, but another one still is needed to fix To be clear, this only happens with de powered doors. If someone at least find the issue, i can try making the second fix later

**Comment by zHonys** [comment]
> Yeah this is basically because there's two reasons a door can close:
> 
>     1. Under its own power
> 
>     2. By prying
> 
> 
> But there's no handling for recognizing the difference between these two states. Door code is complex and this particular route has been missed.
> 
> This needs encapsulating by extending the states a door can be in to prevent this modelling drift. A door can be closing under its own power (Closing) or by an external factor that does not require power (ForceClosing). Encapsulating this will allow for code routes that make sense.

I created a new property called BeingPried, but changing it to BeingForced, so that it can be used for other things as well is a really good ideia, I'll change to this, and maybe change the PryDoor methods to this general usage as well, but i want to fix the main animation bug first
